### PR TITLE
fix(Prime Video): fix durations

### PR DESCRIPTION
**Pull request by mallusrgreatv2**
## Description 
I noticed that the currentTime and duration of what I was watching in Prime Video was actually of a short segment, and not of the full video itself. I fixed that by getting the current time and duration from the html.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/P/Prime Video/presence.ts [hunk 1]
```diff
@@ -54,10 +54,15 @@ presence.on("UpdateData", async () => {
...
-					video.duration
-				);
+				const [unformattedCurrentTime, unformattedDuration] = document
+						.querySelector(".atvwebplayersdk-timeindicator-text")
+						.textContent.trim()
+						.split(" / "),
+					[startTimestamp, endTimestamp] = presence.getTimestamps(
+						timeToSeconds(unformattedCurrentTime),
+						timeToSeconds(unformattedDuration) +
+							timeToSeconds(unformattedCurrentTime)
```
Comment (darkvillager2) [inline websites/P/Prime Video/presence.ts:64]:
```suggestion
						presence.timestampFromFormat(unformattedCurrentTime),
						presence.timestampFromFormat(unformattedDuration) +
							presence.timestampFromFormat(unformattedCurrentTime)
```

File: websites/P/Prime Video/presence.ts [hunk 2]
```diff
@@ -72,10 +77,15 @@ presence.on("UpdateData", async () => {
...
 			} else {
-				const [startTimestamp, endTimestamp] = presence.getTimestamps(
-					video.currentTime,
-					video.duration
-				);
+				const [unformattedCurrentTime, unformattedDuration] = document
+						.querySelector(".atvwebplayersdk-timeindicator-text")
+						.textContent.trim()
+						.split(" / "),
+					[startTimestamp, endTimestamp] = presence.getTimestamps(
```
Comment (darkvillager2) [inline websites/P/Prime Video/presence.ts:84]: Ditto

File: websites/P/Prime Video/presence.ts [hunk 3]
```diff
@@ -113,3 +123,11 @@ presence.on("UpdateData", async () => {
 	if (presenceData.details) presence.setActivity(presenceData);
 	else presence.setActivity();
 });
+function timeToSeconds(timeStr: string) {
```
Comment (darkvillager2) [inline websites/P/Prime Video/presence.ts:126]: Not needed

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

File: websites/P/Prime Video/presence.ts (hunk 2 above)
Comment (mallusrgreatv2) [reply websites/P/Prime Video/presence.ts:84]: what do you mean by this? replace the values directly instead of using a variable?

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**